admin/src/main/resources/io/buoyant/admin/js/combined_client_graph.js

```diff
@@ -49,7 +51,8 @@ var CombinedClientGraph = (function() {
         // where the first values from /metrics are very large [linkerd#485]
         count++;
       } else {
-        var filteredData = Query.filter(query, data.specific);
+        var clients = _.map(routers.clients(routerName), function(client) { return client.label; });
```

Comment:
tioli: you can use the map shorthand here
`_.map(routers.clients(routerName),'label')`

https://lodash.com/docs/4.17.2#map